definitions/features/foreman_proxy.rb

```diff
@@ -26,12 +26,16 @@ def dhcp_curl_cmd
   end
 
   def find_http_error_msg(array_output, curl_http_status)
-    http_line = ''
-    array_output.each do |str|
-      next unless str.include?('HTTP')
-      http_line = str
+    if curl_http_status == 0
+      'Curl error'
```

Comment:
Instead of giving this message, can we give a precise message for user so that he can debug further.
for example - `No valid HTTP response code (either network issues or Connection failed for service)`.